.doks/Makefile

```diff
@@ -14,4 +14,17 @@ deploy_to_doks:
 		--set global.cacheSecret=$${SHARED_PLAY_CRYPTO_SECRET} \
 		--set codacy-api.config.license=$${CODACY_LICENSE} \
 		--set global.codacy.url="http://k8s.dev.codacy.org" \
-		--set global.codacy.backendUrl="http://k8s.dev.codacy.org"
+		--set global.codacy.backendUrl="http://k8s.dev.codacy.org" \
+		--set listener.nfsserverprovisioner.storageClass.name="${RELEASE_NAME}-listener-cache-class" \
```

Comment:
I think we should fix this in the listener so that we dont have to do this here
https://bitbucket.org/qamine/repository-listener/src

Reply:
fixeed